src/components/checklist/MainChecklistBlock.jsx

```diff
@@ -22,6 +22,23 @@ const MainChecklistBlock = (props) => {
 
       if (!loggedUser) {
         arrayOfCheckboxArray = initArrayOfCheckboxes(sections_data);
+        const amountOfAllCheckboxes = sections_data.reduce(
```

Comment:
Why do you change this file? I’ve already fixed all issues here, dont change it back